lib/hackney/cloud/storage.rb

```diff
@@ -33,6 +33,16 @@ def save(letter_html:, uuid:, filename:, metadata:)
         { errors: new_doc.errors.full_messages }
       end
 
+      def read_document(id)
+        doc = document_model.find_by(id: id)
```

Comment:
suggest using `find`

```suggestion
        doc = document_model.find(id)
```

Reply:
actually those are different, your suggestion raises an exception in case the ID is not found, but I'll accept your suggestion  and remove the following `raise 'File does not exist!' if doc.nil?`